hack/image/build-helm-image.sh

```diff
@@ -12,9 +17,9 @@ mkdir -p "$BASEIMAGEDIR"
 
 # build operator binary and base image
 pushd "$BASEIMAGEDIR"
-go run "$ROOTDIR/hack/image/helm/scaffold-helm-image.go"
+GOARCH=$1 go run "$ROOTDIR/hack/image/helm/scaffold-helm-image.go"
 
 mkdir -p build/_output/bin/
-cp $ROOTDIR/build/operator-sdk-dev-x86_64-linux-gnu build/_output/bin/helm-operator
-operator-sdk build $1
+cp $ROOTDIR/build/operator-sdk-dev-${2}-linux-gnu build/_output/bin/helm-operator
+operator-sdk build $3
```

Comment:
iirc, the binary that gets built by the scaffolding code puts that arch string into the binary name. If I didn't have that change, the power cp failed b/c it was looking for `operator-sdk-dev-x86_64-linux-gnu`. 

if/when i modify the double-call of this script to just a `docker tag` on push, we might need one less parameter here, but, I still think we'll need that arch part in the binary name.

Reply:
> the scaffolding code puts that arch string into the binary name

Ah, yes. That makes it more difficult.

Maybe we just remove the platform-specific components of the `build/operator-sdk-dev-x86_64-linux-gnu` make target so that the binary that ends up in `./build` for the `image/build` tasks has the same name regardless of the platform the build is running on?

Reply:
i think that might break release_x86_64 target. Can you confirm?
https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L55

Reply:
I don't _think_ it will. The `build/%` target will use whatever filename you pass it. So I think we can leave the release_x86_64 names alone, and just change this line:

https://github.com/operator-framework/operator-sdk/blob/2c7f779f734998a874d445db00014da100418992/Makefile#L141

to this:

```Makefile
image/build/helm: build/operator-sdk-dev
```

And then update: 

https://github.com/operator-framework/operator-sdk/blob/2c7f779f734998a874d445db00014da100418992/hack/image/build-helm-image.sh#L18

to this:

```sh
cp $ROOTDIR/build/operator-sdk-dev build/_output/bin/helm-operator
```

Reply:
yup that looks like it's right (tested it out)